<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGame</class>
 <widget class="QDialog" name="AddGame">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>339</width>
     <height>106</height>
    </rect>
   </property>
   <property name="title">
    <string>Add game from...</string>
   </property>
   <widget class="QRadioButton" name="radioCustom">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>339</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom Location</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioSteam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>339</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Steam</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="NextButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
